e2e/tests/functional/plugins/notebook/tags.e2e.spec.js

```diff
@@ -139,6 +139,23 @@ test.describe('Tagging in Notebooks @addInit', () => {
         await page.locator('[aria-label="OpenMCT Search"] input[type="search"]').fill('sc');
         await expect(page.locator('[aria-label="Search Result"]')).not.toContainText("Driving");
     });
+
+    test('Can delete objects with tags and neither return in search', async ({ page }) => {
+        await createNotebookEntryAndTags(page);
+        // Delete Notebook
+        await page.locator('button[title="More options"]').click();
+        await page.locator('text=Remove').click();
```

Comment:
I would caution against using `text=` selectors with single words that may match other elements on the page. We should try to narrow this down a bit more:

`li:text("Remove")`
or
`li[title="Remove this object from its containing object."]`

would be more deterministic